scripts/semconv/templates/registry/semantic_attributes.j2

```diff
@@ -0,0 +1,72 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Final
+
+{% set file_name = ctx.output + (ctx.root_namespace | kebab_case | replace('-', '_')) ~ "_attributes.py" -%}
+{{- template.set_file_name(file_name) -}}
+{%- import 'common.j2' as c %}
+
+{% set attributes = ctx.attributes | sort(attribute="name") | list %}
```

Comment:
IIRC attributes are already sorted by name in the semconv JQ filters.

Reply:
good catch! metrics are not sorted though  - https://github.com/open-telemetry/weaver/issues/268